config/module.config.php

```diff
@@ -90,6 +91,8 @@
             'formrange'           => \TwbsHelper\Form\View\Helper\FormRange::class,
             'formselect'          => \TwbsHelper\Form\View\Helper\FormSelect::class,
             'formlabel'           => \TwbsHelper\Form\View\Helper\FormLabel::class,
+            'formfile'            => \TwbsHelper\Form\View\Helper\FormFile::class,
```

Comment:
For what purpose did you return the class FormFile? Bootstrap 5 renders file form elements without any custom decorations needed.

Reply:
Useless you're right